src/clients/pagopa.ts

```diff
@@ -1,63 +1,16 @@
-import {
-  ApiHeaderJson,
-  createFetchRequestForApi,
-  TypeofApiCall
-} from "italia-ts-commons/lib/requests";
 import nodeFetch from "node-fetch";
-
-import {
-  activatePaymentDefaultDecoder,
-  ActivatePaymentT,
-  getActivationStatusDefaultDecoder,
-  GetActivationStatusT,
-  getPaymentInfoDefaultDecoder,
-  GetPaymentInfoT
-} from "../../generated/pagopa-proxy/requestTypes";
+import { Client, createClient } from "../../generated/pagopa-proxy/client";
 
 export function PagoPAClient(
-  baseUrl?: string,
+  baseUrl: string,
   // tslint:disable-next-line:no-any
   fetchApi: typeof fetch = (nodeFetch as any) as typeof fetch
-): {
-  readonly activatePayment: TypeofApiCall<ActivatePaymentT>;
-  readonly getActivationStatus: TypeofApiCall<GetActivationStatusT>;
-  readonly getPaymentInfo: TypeofApiCall<GetPaymentInfoT>;
-} {
-  const options = {
+): Client {
+  return createClient({
+    basePath: "",
```

Comment:
Does this param override the spec defined base path?

Reply:
yes, generated client use the value from the spec as default. This basically tells the client to ignore such value. 
Please note that the full host+basepath has been passed by the constructor, which is not touched by this PR
https://github.com/pagopa/io-backend/blob/33c172eb381fe04e84bee1afd8d8538241cecfbb/src/config.ts#L306